### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.13 esercizio 1 (2 dicembre)

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/calcolo-delle-probabilita#334] [Esercizi M-Z]
<img width="708" alt="image" src="https://github.com/user-attachments/assets/22cb75c3-0fff-44d1-8df4-917b56b38f96">

**Comment by alem1105** [comment]
<img width="998" alt="image" src="https://github.com/user-attachments/assets/292ed8f4-ccda-4eeb-8292-4cd415efa8f2">

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/81abbeae-68cc-4257-a927-d733f8815def)

**Comment by rimaout** [comment]
<img width="992" alt="image" src="https://github.com/user-attachments/assets/cbb92bfa-4989-447c-94f7-40b04e098b30" />
